core/app/models/spree/shipment.rb

```diff
@@ -115,14 +115,15 @@ def tax_category
       selected_shipping_rate.try(:tax_rate).try(:tax_category)
     end
 
-    def refresh_rates
+    def refresh_rates(shipping_method_filter = ShippingMethod::DISPLAY_ON_FRONT_END)
```

Comment:
Assignment Branch Condition size for refresh_rates is too high. [18.55/15]<br>Line is too long. [84/80]